Agoric-CLI.md

```diff
@@ -0,0 +1,87 @@
+## Agoric CLI
+
+You use the Agoric CLI to install dependencies, initialize, deploy, and start Agoric projects. `agoric` takes one of five commands as an argument. **Note**: Required argument names need not be given, only their values. Optional argument names must be given, along with their values.
+
+In general, you want to issue Agoric CLI commands in this order:
+1. `agoric init`
+2. `agoric install`
+3. `agoric start` (Usually with `--reset`)
+4. `agoric deploy`
+
+`agoric init`
+- **Required Arguments**:
+    - `project`: Name of the new project to initialize.
+- **Optional Arguments**:
+    - `--dapp-template <name>`: Use the template specified by `<name>`. Defaults to the demo DApp `dapp-simple-exchange`.
+    - `--dapp-base <base-url>`: Look under this directory for the template. Defaults to `git://github.com/Agoric/`
+- **Function**: Creates a new DApp directory named `<project>` with contents copied from the `dapp-template` argument template.
```

Comment:
I'd prefer to have the **function** lines be the first thing in the description, or even to have them immediately above or below the command description without a label.